include/Transforms/FullLoopUnroll/FullLoopUnroll.td

```diff
@@ -0,0 +1,16 @@
+#ifndef INCLUDE_TRANSFORMS_FULLLOOPUNROLL_FULLLOOPUNROLL_TD_
+#define INCLUDE_TRANSFORMS_FULLLOOPUNROLL_FULLLOOPUNROLL_TD_
+
+include "mlir/Pass/PassBase.td"
+
+def FullLoopUnroll : Pass<"full-loop-unroll"> {
+  let summary = "Fully unroll all loops";
+  let description = [{
+  Scan the IR for affine.for loops and unroll them all.
+  }];
+  let dependentDialects = [
+
+  ];
```

Comment:
nit: did it not complain that the affine dialect wasn't registered for the pass? maybe it's because nothing was anchored on it..? i suppose remove this empty option then

Reply:
I think you only need this when the pass generates new ops from the dependent dialect.

Reply:
Fixed!